draft-private-access-tokens.md

```diff
@@ -80,10 +80,79 @@ separation between the parties involved in the issuance and the redemption
 protocols, it is possible to eliminate this information concentration without
 any functional regressions. This document describes such a protocol.
 
+
 ## Requirements
 
 {::boilerplate bcp14}
 
+
+# Motivation
+
+This section describes classes of use cases where an origin would traditionally
+use a stable unique client identifier for enforcing policy. Hiding these
```

Comment:
```suggestion
use a stable and unique client identifier for enforcing attribute-based policy. Hiding these
```

Attributes are listed below in the use cases and include geo/region, service account existence, etc.